osmaxx-py/config/settings/common.py

```diff
@@ -388,3 +381,44 @@
     # to request access to osmaxx (access_denied page)
     'account_manager_username': env.str('ACCOUNT_MANAGER_USERNAME', default='admin'),
 }
+
+# Celery
+BROKER_URL = env.str('DJANGO_CELERY_BROKER_URL', default='amqp://guest:guest@localhost:5672//')
+CELERY_RESULT_BACKEND = BROKER_URL
+
+# Security - defaults taken from Django 1.8
+SECRET_KEY = env.str("DJANGO_SECRET_KEY", default=None)
+ALLOWED_HOSTS = env.list('DJANGO_ALLOWED_HOSTS', default=[])
+X_FRAME_OPTIONS = env.str('DJANGO_X_FRAME_OPTIONS', default='SAMEORIGIN')
+
+## General SECURE Settings - defaults are not secured!
```

Comment:
What defaults are not secure(d)? Django's defaults? The defaults we set here with `default=`?